### DISCUSSION TITLE
ERROR checking admin err="admin account not found"

### DISCUSSION THREAD

**Discussion by morellik** [vinceanalytics/vince#24] [General]
New installation on Debian 11. after running the command _vince admin_ to create the user and started the server, I get this error:

root@localhost:~# vince serve
2024/12/02 10:20:27 [JOB 1] WAL 000002 stopped reading at offset: (data/000002.log: 0); replayed 0 keys in 0 batches
2024/12/02 10:20:27 INFO unpacking geo database path=vince-data/city.mmdb
2024/12/02 10:20:27 INFO unpacked geo database size_in_mb=108
2024/12/02 10:20:27 INFO loading translation data
2024/12/02 10:20:27 INFO complete loading translation elapsed=46.057µs keys=0
2024/12/02 10:20:27 ERROR checking admin err="admin account not found"

**Comment by gernest** [comment]
You are missing `--adminName` and `--adminPassword`

**Comment by morellik** [comment]
I followed the instructions. After the installation I executed:

vince admin --name acme --password 1234

**Reply by gernest** [reply]
```
vince admin --name acme --password 1234
```

**Reply by morellik** [reply]
I did it

**Reply by gernest** [reply]
try `vince serve --adminName=acme --adminPassword=1234`

**Reply by morellik** [reply]
It works. Thanks

**Reply by gernest** [reply]
you are welcome

**Comment by morellik** [comment]
How can I get my site to load js?
               
       GET http://localhost:8080/js/script.js net::ERR_CONNECTION_REFUSED

I tried to use IP instead of localhost, but I've the same error.

**Reply by gernest** [reply]
for localhost  add this snippet on the `<head>` tag of your html page , replace  `yourdomain.co` with the domain you created on vince.


```
<script defer data-domain="yourdomain.com" src="http://localhost:8080/js/script.local.js"></script>
```

**Reply by morellik** [reply]
I did it, but the script isn't  loaded due to ERR_CONNECTION_REFUSED

**Reply by gernest** [reply]
What is your deployment setup ?

**Reply by morellik** [reply]
I've apache that serves a Joomla site via https protocol

**Reply by morellik** [reply]
Nothing to do?

**Reply by gernest** [reply]
I don't use apache, there is nothing I can do to help.
